src/app/@core/styles/global.scss

```diff
@@ -33,6 +33,20 @@ h6 {
 .content-section {
     margin: 30px auto 100px auto;
     max-width: 1260px;
+    @media only screen and (max-width: 1260px) {
+        border: none;
+        margin: 0;
+    }
+}
+
+.content-section-landing {
```

Comment:
should be 
```
.content-section {
    &__landing {}
}
```

And on the element use both classes - `content-section content-section__landing`
This way we have less css transpiled.